bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/StringUtils.java

```diff
@@ -133,4 +133,8 @@ public static StringValue fromString(String s) {
         return new NonBmpStringValue(s, highSurrogatesArr);
     }
 
+    public static BString fromString(BString s) {
```

Comment:
`fromString` is used to convert java string to ballerina. do we need a way to convert BString to BString?